src/core/purify.js

```diff
@@ -7,9 +7,9 @@ export default purify;
 
 purify.addHook('afterSanitizeAttributes', (node) => {
   // Set all elements owning target to target=_blank
-  // and add rel="noreferrer".
+  // and add rel="noopener noreferrer".
   if ('target' in node) {
     node.setAttribute('target', '_blank');
-    node.setAttribute('rel', 'noreferrer');
+    node.setAttribute('rel', 'noopener noreferrer');
```

Comment:
From what I [can read about the noopener attack](https://mathiasbynens.github.io/rel-noopener/) norefferer is for older browsers.